### DISCUSSION TITLE
Can I use both persist and subscribe with selector middleware together in the same store?

### DISCUSSION THREAD

**Discussion by chrissysemens** [pmndrs/zustand#1892] [General] [answered]
As discussion, I'd like to be persisting data to localStorage whilst being able to subscribe to the stores changes, is this possible?

**Comment by dai-shi** [comment, marked as answer]
It should.
https://github.com/pmndrs/zustand/blob/9333eb629c4971159e839311bdca03233ad4e65f/tests/middlewareTypes.test.tsx#L541-L550

You want to use `create<MyState>()(...)` style instead.

**Reply by chrissysemens** [reply]
Perfect <3
